arkd.Dockerfile

```
@@ -9,8 +9,8 @@ WORKDIR /app
 RUN git clone https://github.com/arkade-os/arkd.git
```

let's change also this step and ensure we clone exactly the v0.7.0 tag:
```suggestion
RUN git clone --branch v0.7.0 --single-branch https://github.com/arkade-os/arkd.git
```